### PR TITLE
Add a helper function that allows easier generation of board layout config.

### PR DESCRIPTION
**Pull request by tomprince**
You can run via `lua GenerateMapData()` in the console, and it will generate a new notebook page with code you can copy.

### CODE REVIEW

**Comment by tomprince** [comment]
I've rebased this on #25 to use the new data format.

File: global-script.lua [hunk 1]
```diff
@@ -2030,357 +2029,232 @@ scaleFactors = {
...
+            { pos = Vector(-10.34, 1.08, 18.04), rot = Vector(0.00, 180.00, 0.00), board = "W" },
         },
         ["Top to Top"] = {
-            Vector(9.13, 1.08, 25.29),
-            Vector(-0.06, 1.08, 9.62),
+            { pos = Vector(9.13, 1.08, 25.29), rot = Vector(0.00, 0.00, 0.00) },
+            { pos = Vector(-0.06, 1.08, 9.62), rot = Vector(0.00, 180.00, 0.00) },
         },
         ["Coastline"] = {
-            Vector(20.38, 1.08, 9.96),
```
Comment (iakona) [inline global-script.lua:2056]: why are these lands being flipped around?

File: global-script.lua [hunk 2]
```diff
@@ -2030,357 +2029,232 @@ scaleFactors = {
...
         ["Fragment"] = {
-            Vector(-5.20, 1.08, 18.87),
-            Vector(10.12, 1.08, 19.08),
+            { pos = Vector(-5.20, 1.08, 18.87), rot = Vector(0.00, 90.00, 0.00) },
+            { pos = Vector(10.12, 1.08, 19.08), rot = Vector(0.00, 330.00, 0.00) },
         },
-        ["Inverted Fragment"] = {
-            Vector(-5.44, 1.08, 18.99),
-            Vector(10.12, 1.08, 19.08),
+        ["Fragment 2"] = {
```
Comment (iakona) [inline global-script.lua:2066]: this should stay as `Inverted Fragment` there's code in json load to auto convert Fragment 2 into Inverted Fragment

**Comment by iakona** [comment]
this PR doesn't seem to be up to date

**Comment by tomprince** [comment]
Rebased.

**Review by iakona — APPROVED** [review]

**Pull request merged**